library/ix-dev/community/grafana/templates/_grafana.tpl

```diff
@@ -0,0 +1,124 @@
+{{- define "grafana.workload" -}}
+workload:
+  grafana:
+    enabled: true
+    primary: true
+    type: Deployment
+    podSpec:
+      hostNetwork: {{ .Values.grafanaNetwork.hostNetwork }}
+      containers:
+        grafana:
+          enabled: true
+          primary: true
+          imageSelector: image
+          securityContext:
+            runAsUser: {{ .Values.grafanaRunAs.user }}
+            runAsGroup: {{ .Values.grafanaRunAs.group }}
+          envFrom:
+            - configMapRef:
+                name: grafana-config
+          {{ with .Values.grafanaConfig.additionalEnvs }}
+          envList:
+            {{ range $env := . }}
+            - name: {{ $env.name }}
+              value: {{ $env.value }}
+            {{ end }}
+          {{ end }}
+          probes:
+            {{ $scheme := "http" }}
+            {{ if .Values.grafanaNetwork.certificateID }}
+              {{ $scheme = "https" }}
+            {{ end }}
+            liveness:
+              enabled: true
+              type: {{ $scheme }}
+              port: {{ .Values.grafanaNetwork.webPort }}
+              path: /api/health
+            readiness:
+              enabled: true
+              type: {{ $scheme }}
+              port: {{ .Values.grafanaNetwork.webPort }}
+              path: /api/health
+            startup:
+              enabled: true
+              type: {{ $scheme }}
+              port: {{ .Values.grafanaNetwork.webPort }}
+              path: /api/health
+      initContainers:
+      {{- include "ix.v1.common.app.permissions" (dict "containerName" "01-permissions"
+                                                        "UID" .Values.grafanaRunAs.user
```

Comment:
The permission container will run as the grafana user?

Reply:
Nop, it runs as root (hardcoded), this value here is used in the script that fixes the perms/ownership